ir_filter_security_extra/security/ir_filter_security_extra.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <record id="group_ir_filters_manager" model="res.groups" >
+            <field name="name">Filters Manager</field>
+            <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
+            <field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"/>
```

Comment:
This is already achieved, so it's not required IMO due to https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/odoo/addons/base/security/base_security.xml#L107 and https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/odoo/addons/base/security/base_groups.xml#L14-L17